river/datasets/synth/sea.py

```diff
@@ -6,11 +6,19 @@
 class SEA(base.SyntheticDataset):
     """SEA synthetic dataset.
 
-    Each observation is composed of 3 features. Only the first two features are relevant. The
+    Implementation of the data stream with abrupt drift described in [^1]. Each observation is composed of 3 features. Only the first two features are relevant. The
```

Comment:
Is the line character limit per line being exceeded here?

Reply:
Nice catch. Fixed in 27ef120bd2b0d3c4e1560e223db91ffba0415b16